allure-android/src/main/kotlin/ru/tinkoff/allure/android/AllureAndroidRunner.kt

```diff
@@ -12,7 +13,14 @@ import android.support.test.runner.AndroidJUnitRunner
 @Keep
 open class AllureAndroidRunner : AndroidJUnitRunner() {
     override fun onCreate(arguments: Bundle) {
-        arguments.putCharSequence("listener", AllureAndroidListener::class.java.name)
+        val oldListeners: String? = arguments.getString("listener")
+        val listenersStringBuilder = StringBuilder()
+        if (oldListeners != null) {
+            listenersStringBuilder.append(oldListeners)
+            listenersStringBuilder.append(",")
+        }
+        listenersStringBuilder.append(AllureAndroidListener::class.java.name)
+        arguments.putCharSequence("listener", listenersStringBuilder.toString())
```

Comment:
it can be written more kotlin-way, e. g:
```kotlin
arguments.putCharSequence("listener", arguments.getCharSequence("listener")
        ?.let {
            "$it,${AllureAndroidListener::class.java.name}"
        }
        ?: AllureAndroidListener::class.java.name)
```